addons/test_mail_full/tests/test_sms_composer.py

```diff
@@ -148,6 +149,18 @@ def test_composer_default_recipient(self):
         self.assertFalse(composer.recipient_single_valid)
         self.assertEqual(composer.recipient_single_description, self.test_record.customer_id.display_name)
 
+    def test_composer_nofield_w_customer(self):
```

Comment:
What are we testing here ? Missing docstring, otherwise we don't know what it is testing.